### PR TITLE
Map display property to checkbox label class name attribute

### PR DESCRIPTION
**Pull request by JoeChapman**
- add conditional display class name in checkbox partial
- add display value to opts and mixin to checkbox options

### CODE REVIEW

**Comment by gavboulton** [comment]
As part of govuk_elements you can add `inline` to the parent of `block-label`s so that they display inline, https://github.com/alphagov/govuk_elements/blob/master/public/sass/elements/forms/_form-block-labels.scss#L47-L51.

It's probably better to add an optional `className` to the group (after compound), which will then be more flexible for other uses.

**Comment by easternbloc** [comment]
@gavboulton I'm struggling to see how this works? We have an issue where we'd like to be able to remove the default block-label styling that's currently enforced using the template. Is there a way to remove that styling using the toolkit?

**Comment by gavboulton** [comment]
@easternbloc There's not currently a way to remove `block-label`, but this PR adds the possibility to add `something-label`, not completely remove it. When looking at the tests I thought the idea was to inline the elements.

If the you could change `display` to `className` and default it to `block-label` and remove 'label' from the view then I think this should be good to go.

**Comment by easternbloc** [comment]
Cool sounds good @gavboulton will do.

File: partials/forms/checkbox.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div id="{{key}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{#error}} validation-error{{/error}}">
     {{#error}}<p class="error-message" aria-hidden="true">{{error.message}}</p>{{/error}}
-    <label for="{{key}}" class="block-label{{#invalid}} invalid-input{{/invalid}}">
+    <label for="{{key}}" class="{{#className}} {{className}}{{/className}}{{#invalid}}invalid-input{{/invalid}}">
```
Comment (lennym) [inline partials/forms/checkbox.html]: The whitespace here will result in the `{{className}}` and `invalid-input` being run together. 

`{{#className}} {{className}}{{/className}}` should be `{{#className}}{{className}} {{/className}}`

**Comment by lennym** [comment]
Published @v1.1.0

**Pull request merged**